restorers/model/nafnet/nafblock.py

```diff
@@ -12,6 +12,10 @@ class SimpleGate(keras.layers.Layer):
     """
     Simple Gate
     It splits the input of size (b,h,w,c) into tensors of size (b,h,w,c//factor) and returns their Hadamard product
+
+    Reference: NAFNet Paper (Simple Baselines for Image Restoration)
+    https://www.ecva.net/papers/eccv_2022/papers_ECCV/papers/136670017.pdf
```

Comment:
I am sorry but is there a `abs` version of the paper? I don't like it when I click on a link and it starts downloading a paper.

If there is none, we can proceed with this.